app/resolution.py

```diff
@@ -14,12 +14,12 @@
 
 
 class ResolutionFinder:
-    DEFAULT_MODEL_URL = "https://drive.google.com/u/0/uc?id=15v3L5-tcCeDpTQFj6qHtecmXuYssaNnA&export=download"
+    DEFAULT_MODEL_URL = "https://drive.google.com/u/0/uc?id=1i9tb42YtHwdnP5474YdsDuCQn2IfbX6J&export=download"
```

Comment:
The model, how it was trained, and the dataset should be open sourced too

Reply:
Model was trained here: https://colab.research.google.com/drive/1Y0Y_-xeoku9xGLl-6Tpi8DYuOUxtSJXH?usp=sharing
Unfortunately, it was hard to achieve satisfactory accuracy with the rather low resources that colab offers.

As for the dataset, @mrojew  created such a program to generate it.
https://github.com/mrojew/ImageDecoder?fbclid=IwAR0eYp8xfmxOfMkpj3_AfIFaidnavCRAzd1PNxII9Pw4F9Idr5jK-1dBXy8